src/common/rbac/project/util.go

```diff
@@ -56,17 +76,25 @@ var (
 		{Resource: rbac.ResourceReplicationJob, Action: rbac.ActionList},
 
 		{Resource: rbac.ResourceLabel, Action: rbac.ActionCreate},
+		{Resource: rbac.ResourceLabel, Action: rbac.ActionRead},
 		{Resource: rbac.ResourceLabel, Action: rbac.ActionUpdate},
 		{Resource: rbac.ResourceLabel, Action: rbac.ActionDelete},
 		{Resource: rbac.ResourceLabel, Action: rbac.ActionList},
 
+		{Resource: rbac.ResourceLabelResource, Action: rbac.ActionList},
+
 		{Resource: rbac.ResourceRepository, Action: rbac.ActionCreate},
+		{Resource: rbac.ResourceRepository, Action: rbac.ActionRead},
 		{Resource: rbac.ResourceRepository, Action: rbac.ActionUpdate},
 		{Resource: rbac.ResourceRepository, Action: rbac.ActionDelete},
 		{Resource: rbac.ResourceRepository, Action: rbac.ActionList},
-		{Resource: rbac.ResourceRepository, Action: rbac.ActionPushPull}, // compatible with security all perm of project
-		{Resource: rbac.ResourceRepository, Action: rbac.ActionPush},
 		{Resource: rbac.ResourceRepository, Action: rbac.ActionPull},
+		{Resource: rbac.ResourceRepository, Action: rbac.ActionPush},
+		{Resource: rbac.ResourceRepository, Action: rbac.ActionPushPull}, // compatible with security all perm of project
```

Comment:
Do we still need this?

Reply:
ISSUE https://github.com/goharbor/harbor/issues/6881